dashboard/app/controllers/pd/workshop_daily_survey_controller.rb

```diff
@@ -410,6 +447,29 @@ def render_csf_survey(survey_name, workshop)
       render :new_general
     end
 
+    def render_csf_survey_foorm(survey_name, workshop)
+      form, latest_version = Foorm::Form.get_form_and_latest_version_for_name(survey_name)
+      form_questions = JSON.parse(form.questions)
+
+      @script_data = {
+        props: {
+          formQuestions: form_questions,
+          formName: survey_name,
+          formVersion: latest_version,
+          surveyData: {
```

Comment:
you don't need workshop_course unless you need that parameter in your survey (or you want it in the answers submitted)